### PR TITLE
Attention, CNN, DLC, LN, MLP & Intro Full Scan

### PR DESCRIPTION
**Pull request by goldmermaid**
"Full Scan" meaning: @astonzhang 

1. Code:
    1. Import in alphabetical order
    2. 78 chars per line with ruler
    3. Lint

2. Text:
    1. Title capitalization & Verb to noun
    2. Figure referring
    3. Spell typo fixing

### CODE REVIEW

**Comment by astonzhang** [comment]
what do you mean by full scan? only lint?

File: chapter_linear-networks/fashion-mnist.md [hunk 1]
```diff
@@ -133,15 +134,15 @@ for X, y in train_iter:
 '%.2f sec' % timer.stop()
 ```
 
-## Put all Things Together 
+## Putting Things Together
```
Comment (astonzhang) [inline chapter_linear-networks/fashion-mnist.md]: mytodo, putting all

File: chapter_linear-networks/linear-regression-scratch.md [hunk 2]
```diff
@@ -105,7 +105,8 @@ def data_iter(batch_size, features, labels):
     # The examples are read at random, in no particular order
     random.shuffle(indices)
     for i in range(0, num_examples, batch_size):
-        batch_indices = np.array(indices[i: min(i + batch_size, num_examples)])
+        batch_indices = \
```
Comment (astonzhang) [inline chapter_linear-networks/linear-regression-scratch.md]: mytodo, break from parenthese

Comment (goldmermaid) [reply chapter_linear-networks/linear-regression-scratch.md]: i see

**Pull request merged**